av/video/stream.pyx

```diff
@@ -10,3 +10,13 @@ cdef class VideoStream(Stream):
             self.codec_context.height,
             id(self),
         )
+
+    def __getattr__(self, name):
```

Comment:
Uhoh, getattr / setattr have already bitten us many times, why do we need this?

Reply:
This blocks pass through access to `framerate` from the existing getattr/setattr methods in the Stream class. We can also do this by making an explicit `framerate` property here instead of overriding the getattr/setattr methods.
However, maybe we can just remove the passthrough access altogether. It's a larger breaking change, but I just tried it and it doesn't seem to break the tests. What do you think?

Reply:
Whoops I lied, it doesn't work

Reply:
I'm really not comfortable with adding any more `__setattr__` or `__getattr__` dunder methods. If the root cause of our issue is the existing dunder methods in `Stream`, yes by all means block access there with an explicit comment stating why we are doing it. A unit test which checks an AttributeError is raised on get/set seems necessary too.

Reply:
Sure, that sounds fine to me. I'll just remove any special treatment. However, that means #1005 still stands, and any existing code that relies on VideoStream.framerate may silently fail.

Reply:
How about raising an `AttributeError` when accessing `framerate` in the existing getattr/setattr of `Stream`? That avoids adding more dunder methods and avoids the issues with adding a separate property.

Reply:
I'll open that as a separate PR so we can evaluate it separately.